src/device-registry/utils/create-site.js

```diff
@@ -222,6 +224,13 @@ const createSite = {
           }
           if (isEmpty(responseJSON.data)) {
             logElement("unable to list stations");
+            return {
```

Comment:
Hi @BenjaminSsempala , can you shed some light on this specific change under` listWeatherStations`?

1. Can we ensure that it does not break anything else?
2. What other functions utilise it? Do they anticipate this addition in the way they respond to the output from this function?

Thanks!

Reply:
Thanks @Baalmart 
I've confirmed this